2025-01-31 21:48:06,125:DEBUG:certbot._internal.main:certbot version: 3.1.0
2025-01-31 21:48:06,135:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/local/bin/certbot
2025-01-31 21:48:06,135:DEBUG:certbot._internal.main:Arguments: ['--webroot', '--webroot-path=/var/www/certbot', '--email', 'karimmaroous@gmail.com', '--agree-tos', '--no-eff-email', '--staging', '-d', 'opsforall.net']
2025-01-31 21:48:06,136:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2025-01-31 21:48:06,164:DEBUG:certbot._internal.log:Root logging level set at 30
2025-01-31 21:48:06,165:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2025-01-31 21:48:06,167:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Saves the necessary validation files to a .well-known/acme-challenge/ directory within the nominated webroot path. A separate HTTP server must be running and serving files from the webroot path. HTTP challenge only (wildcards not supported).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='webroot', value='certbot._internal.plugins.webroot:Authenticator', group='certbot.plugins')
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x76a82c794350>
Prep: True
2025-01-31 21:48:06,167:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x76a82c794350> and installer None
2025-01-31 21:48:06,168:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2025-01-31 21:48:06,338:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-staging-v02.api.letsencrypt.org/acme/acct/182784314', new_authzr_uri=None, terms_of_service=None), 5c0ea7419c19db18f607881e60bb62df, Meta(creation_dt=datetime.datetime(2025, 1, 31, 21, 18, 24, tzinfo=datetime.timezone.utc), creation_host='f5a16ff59e49', register_to_eff=None))>
2025-01-31 21:48:06,339:DEBUG:acme.client:Sending GET request to https://acme-staging-v02.api.letsencrypt.org/directory.
2025-01-31 21:48:06,345:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-staging-v02.api.letsencrypt.org:443
2025-01-31 21:48:06,523:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 974
2025-01-31 21:48:06,523:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 21:48:06 GMT
Content-Type: application/json
Content-Length: 974
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "keyChange": "https://acme-staging-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "profiles": {
      "classic": "The same profile you're accustomed to",
      "tlsserver": "https://letsencrypt.org/2025/01/09/acme-profiles/"
    },
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.4-April-3-2024.pdf",
    "website": "https://letsencrypt.org/docs/staging-environment/"
  },
  "newAccount": "https://acme-staging-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-staging-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-staging-v02.api.letsencrypt.org/acme/new-order",
  "renewalInfo": "https://acme-staging-v02.api.letsencrypt.org/draft-ietf-acme-ari-03/renewalInfo",
  "revokeCert": "https://acme-staging-v02.api.letsencrypt.org/acme/revoke-cert",
  "z6G2pljXig0": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417"
}
2025-01-31 21:48:06,524:DEBUG:certbot._internal.display.obj:Notifying user: Requesting a certificate for opsforall.net
2025-01-31 21:48:06,530:DEBUG:acme.client:Requesting fresh nonce
2025-01-31 21:48:06,531:DEBUG:acme.client:Sending HEAD request to https://acme-staging-v02.api.letsencrypt.org/acme/new-nonce.
2025-01-31 21:48:06,590:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2025-01-31 21:48:06,592:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 21:48:06 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: N9Ep_yiTELxeDFElDoabDh71tYfJrnGcJnHca38u16kxZ9BauUU
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2025-01-31 21:48:06,595:DEBUG:acme.client:Storing nonce: N9Ep_yiTELxeDFElDoabDh71tYfJrnGcJnHca38u16kxZ9BauUU
2025-01-31 21:48:06,595:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "opsforall.net"\n    }\n  ]\n}'
2025-01-31 21:48:06,598:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xODI3ODQzMTQiLCAibm9uY2UiOiAiTjlFcF95aVRFTHhlREZFbERvYWJEaDcxdFlmSnJuR2NKbkhjYTM4dTE2a3haOUJhdVVVIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "OMg57tLgcSNelNZk3GH8CIUba_EDweKf1GomfkGUswsuTa3CoY7O5JGPukeUuG0mVY7ViWcMCoCKQzD5zxVSa9mSUCl_qzV2mVXakxTtZVbq4BIXVhQ1pHplxwI3ww9QIhMungbnaUFUJnFym97rXGVFJLXenm4ZeO76gHUrF3J06gH_2rhpfU2HDCRJmq4YvfAKYJPmfw_3al7hbFN-Ya-ZT41KC9fVxyQHelD4hd_O21DzbUGQScbE95K1Q7yPolaFgjU5FRGRuV9ANEhYawjx0BeesO3L7eVQz9GfZ_rp90aa9tBOUMJYGP_G62pF5sM3rms5w5PIn90D3AZVZw",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogIm9wc2ZvcmFsbC5uZXQiCiAgICB9CiAgXQp9"
}
2025-01-31 21:48:06,685:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 359
2025-01-31 21:48:06,686:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Fri, 31 Jan 2025 21:48:06 GMT
Content-Type: application/json
Content-Length: 359
Connection: keep-alive
Boulder-Requester: 182784314
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-staging-v02.api.letsencrypt.org/acme/order/182784314/22333569634
Replay-Nonce: xMYr_xat7dDHoUtW1nsWjhh7ksOu3Pyw55i8UtPGQrA6oXCQgZE
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2025-02-07T21:48:06Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "opsforall.net"
    }
  ],
  "authorizations": [
    "https://acme-staging-v02.api.letsencrypt.org/acme/authz/182784314/15902310834"
  ],
  "finalize": "https://acme-staging-v02.api.letsencrypt.org/acme/finalize/182784314/22333569634"
}
2025-01-31 21:48:06,689:DEBUG:acme.client:Storing nonce: xMYr_xat7dDHoUtW1nsWjhh7ksOu3Pyw55i8UtPGQrA6oXCQgZE
2025-01-31 21:48:06,689:DEBUG:acme.client:JWS payload:
b''
2025-01-31 21:48:06,698:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/authz/182784314/15902310834:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xODI3ODQzMTQiLCAibm9uY2UiOiAieE1Zcl94YXQ3ZERIb1V0VzFuc1dqaGg3a3NPdTNQeXc1NWk4VXRQR1FyQTZvWENRZ1pFIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzE4Mjc4NDMxNC8xNTkwMjMxMDgzNCJ9",
  "signature": "Mfc90u0Vh9lJBMcYM0HXg_aIy8qlzWiddoj7NMyHE1Q1-v9AuAmCyaGCWkIN2ApxHa_ZE6OhUVyl6mkGm0Ucr1efopKMQVBbsydi9h16vonT4HdfW67KQaPdfvTbGm86--e8vQKNrmZlm7eTQ9OMs-P1-6oOIX1MjQDkMuU0h-Iu-lYg3vBpnLRlr3SoRniZ6UCDgWsAW1g_lgYZo0dg7oJ7wk5SpgyRLl47p2J3_oU-Pr3F36CChZxt3dNNZPjXINX52FqQesSPxKGf0vSvP-ISRADYh0Q-hZ41WijFtdwMKdieBh4hC6TQbTobF0nbaKEO50kr9Ry638hyLc72eQ",
  "payload": ""
}
2025-01-31 21:48:06,766:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/authz/182784314/15902310834 HTTP/1.1" 200 839
2025-01-31 21:48:06,769:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 21:48:06 GMT
Content-Type: application/json
Content-Length: 839
Connection: keep-alive
Boulder-Requester: 182784314
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: xMYr_xatxtpP2l2-Z7qVMEXp3hkseWjErmt9jbC8uTe2z0FGQlc
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "opsforall.net"
  },
  "status": "pending",
  "expires": "2025-02-07T21:48:06Z",
  "challenges": [
    {
      "type": "http-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902310834/MCtuDg",
      "status": "pending",
      "token": "0KO6HL1cujqp8kl761RmlH7xEgzAkI1cpTtGSiuGIhE"
    },
    {
      "type": "dns-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902310834/fIVs7w",
      "status": "pending",
      "token": "0KO6HL1cujqp8kl761RmlH7xEgzAkI1cpTtGSiuGIhE"
    },
    {
      "type": "tls-alpn-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902310834/LR9WFA",
      "status": "pending",
      "token": "0KO6HL1cujqp8kl761RmlH7xEgzAkI1cpTtGSiuGIhE"
    }
  ]
}
2025-01-31 21:48:06,770:DEBUG:acme.client:Storing nonce: xMYr_xatxtpP2l2-Z7qVMEXp3hkseWjErmt9jbC8uTe2z0FGQlc
2025-01-31 21:48:06,771:INFO:certbot._internal.auth_handler:Performing the following challenges:
2025-01-31 21:48:06,771:INFO:certbot._internal.auth_handler:http-01 challenge for opsforall.net
2025-01-31 21:48:06,771:INFO:certbot._internal.plugins.webroot:Using the webroot path /var/www/certbot for all unmatched domains.
2025-01-31 21:48:06,771:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /var/www/certbot/.well-known/acme-challenge
2025-01-31 21:48:06,772:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /var/www/certbot/.well-known/acme-challenge/0KO6HL1cujqp8kl761RmlH7xEgzAkI1cpTtGSiuGIhE
2025-01-31 21:48:06,772:DEBUG:acme.client:JWS payload:
b'{}'
2025-01-31 21:48:06,776:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902310834/MCtuDg:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xODI3ODQzMTQiLCAibm9uY2UiOiAieE1Zcl94YXR4dHBQMmwyLVo3cVZNRVhwM2hrc2VXakVybXQ5amJDOHVUZTJ6MEZHUWxjIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2NoYWxsLzE4Mjc4NDMxNC8xNTkwMjMxMDgzNC9NQ3R1RGcifQ",
  "signature": "SDYC6TDlP7ZX-2RM829b9ETmF3LGnar2o7ZeErI02yBSREjijh1p7yjjbLP59AjZpP70Sj3mjmkX8V-gEllF1tFDTVIrBRtg9woPRCCWNZQAtH3uc1MBpqBmHi3s-plD8EgErcd3u69_ZR07bUpnrPL4o6w9Oq-QQSDDIusMkIImZwH6SnpVUQTEBmZIosQu6_AE-Ij1atEXUAbrW_WUCEEQY6odJrjNcXqzW3z1ni_wKjrvPqoIbuwo4Qqgsi6qHA-lRAOeSwlzgOYllEG67nPkOJATUC6MV1E8MYePEu_EwS6e7Bddr6dDQfP_yiRIsCVGs3zNGiI2TCIolo9dLQ",
  "payload": "e30"
}
2025-01-31 21:48:06,846:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/chall/182784314/15902310834/MCtuDg HTTP/1.1" 200 201
2025-01-31 21:48:06,847:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 21:48:06 GMT
Content-Type: application/json
Content-Length: 201
Connection: keep-alive
Boulder-Requester: 182784314
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-staging-v02.api.letsencrypt.org/acme/authz/182784314/15902310834>;rel="up"
Location: https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902310834/MCtuDg
Replay-Nonce: N9Ep_yiTH7UJYJuka6v7pkK5Mtt5ByhtUUj92HhwAW8dRUbwZYM
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902310834/MCtuDg",
  "status": "pending",
  "token": "0KO6HL1cujqp8kl761RmlH7xEgzAkI1cpTtGSiuGIhE"
}
2025-01-31 21:48:06,847:DEBUG:acme.client:Storing nonce: N9Ep_yiTH7UJYJuka6v7pkK5Mtt5ByhtUUj92HhwAW8dRUbwZYM
2025-01-31 21:48:06,847:INFO:certbot._internal.auth_handler:Waiting for verification...
2025-01-31 21:48:07,848:DEBUG:acme.client:JWS payload:
b''
2025-01-31 21:48:07,851:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/authz/182784314/15902310834:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xODI3ODQzMTQiLCAibm9uY2UiOiAiTjlFcF95aVRIN1VKWUp1a2E2djdwa0s1TXR0NUJ5aHRVVWo5Mkhod0FXOGRSVWJ3WllNIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzE4Mjc4NDMxNC8xNTkwMjMxMDgzNCJ9",
  "signature": "szgK4DY2Dfzxe81IuC5JhygiTPkvDneCEqZINeG3L30FONUzH7L3CdqpJVtSIrJZDOSesarEO9BeoYd-5FB-Aob1c83FIG2UDCq8mBDlYoJ5QWlcnp7mcQWpK4DsbJII6ti2_gOR8NZ_xCu8_I9Ff-YGwng2O0wmxzKnpOJkS8CkK57nJzfngYPVISuQXUyy1RbKYK6BOfxKMZrDf08gS4DHUFebWnbaZz0yn37tlvAdhwEkLlajJZIClDVOBUega2j0h-ivgYh59WW4fha7uVCcp3CGaVwscR34vhRyhCzwHUCLZs6PV5XuL9_zEJlopJlomsy0idEb35agOBvIOg",
  "payload": ""
}
2025-01-31 21:48:07,912:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/authz/182784314/15902310834 HTTP/1.1" 200 1031
2025-01-31 21:48:07,912:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 21:48:07 GMT
Content-Type: application/json
Content-Length: 1031
Connection: keep-alive
Boulder-Requester: 182784314
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: xMYr_xat_TnBPaCf7zLWihXa1hYXnU0ayDcj7hYoJER6BOF4DCo
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "opsforall.net"
  },
  "status": "invalid",
  "expires": "2025-02-07T21:48:06Z",
  "challenges": [
    {
      "type": "http-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902310834/MCtuDg",
      "status": "invalid",
      "validated": "2025-01-31T21:48:06Z",
      "error": {
        "type": "urn:ietf:params:acme:error:connection",
        "detail": "54.226.228.6: Fetching http://opsforall.net/.well-known/acme-challenge/0KO6HL1cujqp8kl761RmlH7xEgzAkI1cpTtGSiuGIhE: Connection refused",
        "status": 400
      },
      "token": "0KO6HL1cujqp8kl761RmlH7xEgzAkI1cpTtGSiuGIhE",
      "validationRecord": [
        {
          "url": "http://opsforall.net/.well-known/acme-challenge/0KO6HL1cujqp8kl761RmlH7xEgzAkI1cpTtGSiuGIhE",
          "hostname": "opsforall.net",
          "port": "80",
          "addressesResolved": [
            "54.226.228.6"
          ],
          "addressUsed": "54.226.228.6"
        }
      ]
    }
  ]
}
2025-01-31 21:48:07,912:DEBUG:acme.client:Storing nonce: xMYr_xat_TnBPaCf7zLWihXa1hYXnU0ayDcj7hYoJER6BOF4DCo
2025-01-31 21:48:07,913:INFO:certbot._internal.auth_handler:Challenge failed for domain opsforall.net
2025-01-31 21:48:07,913:INFO:certbot._internal.auth_handler:http-01 challenge for opsforall.net
2025-01-31 21:48:07,913:DEBUG:certbot._internal.display.obj:Notifying user: 
Certbot failed to authenticate some domains (authenticator: webroot). The Certificate Authority reported these problems:
  Domain: opsforall.net
  Type:   connection
  Detail: 54.226.228.6: Fetching http://opsforall.net/.well-known/acme-challenge/0KO6HL1cujqp8kl761RmlH7xEgzAkI1cpTtGSiuGIhE: Connection refused

Hint: The Certificate Authority failed to download the temporary challenge files created by Certbot. Ensure that the listed domains serve their content from the provided --webroot-path/-w and that files created there can be downloaded from the internet.

2025-01-31 21:48:07,915:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2025-01-31 21:48:07,915:DEBUG:certbot._internal.error_handler:Calling registered functions
2025-01-31 21:48:07,915:INFO:certbot._internal.auth_handler:Cleaning up challenges
2025-01-31 21:48:07,915:DEBUG:certbot._internal.plugins.webroot:Removing /var/www/certbot/.well-known/acme-challenge/0KO6HL1cujqp8kl761RmlH7xEgzAkI1cpTtGSiuGIhE
2025-01-31 21:48:07,915:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2025-01-31 21:48:07,915:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/local/bin/certbot", line 33, in <module>
    sys.exit(load_entry_point('certbot', 'console_scripts', 'certbot')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/main.py", line 19, in main
    return internal_main.main(cli_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1873, in main
    return config.func(config, plugins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1579, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 142, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 518, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 429, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 497, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, self.config, best_effort)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2025-01-31 21:48:07,918:ERROR:certbot._internal.log:Some challenges have failed.
